test/unit/ngsiv2/HTTP_reveice_measures-test.js

```diff
@@ -0,0 +1,329 @@
+            .post('/v1/updateContext')
+            .reply(200, utils.readExampleFile('./test/contextResponses/multipleMeasuresSuccess.json'));
+
+        iotagentMqtt.start(config, function() {
+            request(provisionOptions, function(error, response, body) {
+                done();
+            });
+        });
+    });
+
+    afterEach(function(done) {
+        nock.cleanAll();
+
+        async.series([
+            iotAgentLib.clearAll,
+            iotagentMqtt.stop
+        ], done);
+    });
+
+    describe('When a POST measure arrives for the HTTP binding', function() {
+        var optionsMeasure = {
+            url: 'http://localhost:' + config.http.port + '/iot/json',
+            method: 'POST',
+            json: {
+                humidity: '32',
+                temperature: '87'
+            },
+            headers: {
+                'fiware-service': 'smartGondor',
+                'fiware-servicepath': '/gardens'
+            },
+            qs: {
+                i: 'MQTT_2',
+                k: '1234'
+            }
+        };
+
+        beforeEach(function() {
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post('/v2/entities/Second%20MQTT%20Device/attrs',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/multipleMeasures.json'))
+                .reply(204);
+        });
+        it('should return a 200 OK with no error', function(done) {
+            request(optionsMeasure, function(error, result, body) {
+                should.not.exist(error);
+                result.statusCode.should.equal(200);
+                done();
+            });
+        });
+
+        it('should send its value to the Context Broker', function(done) {
+            request(optionsMeasure, function(error, result, body) {
+                contextBrokerMock.done();
+                done();
+            });
+        });
+
+    });
+
+    describe('When a POST measure arrives with a TimeInstant attribute in the body', function() {
+        var optionsMeasure = {
+                url: 'http://localhost:' + config.http.port + '/iot/json',
+                method: 'POST',
+                json: {
+                    humidity: '111222',
+                    TimeInstant: '20200222T222222'
+                },
+                headers: {
+                    'fiware-service': 'smartGondor',
+                    'fiware-servicepath': '/gardens'
+                },
+                qs: {
+                    i: 'dev0130101',
+                    k: '1234'
+                }
+            },
+            provisionOptions = {
+                url: 'http://localhost:' + config.iota.server.port + '/iot/devices',
+                method: 'POST',
+                json: utils.readExampleFile('./test/deviceProvisioning/provisionDeviceTimeinstant.json'),
```

Comment:
Which is the role of the `"expression": "${@humedad / 1000}",` in the JSON file used for provision in this case? It seems to be orthoghonal to the test...

Reply:
I guess that this is an error because there is no `humedad` attribute. Again, we have inherited this from the corresponding NGSIv1 version. Probably they intended to test the expression plugin and for some reason... they changed their mind! :)

Reply:
Prio to this PR is seems that .json file is used only from a single test file NGSIv1 (https://github.com/telefonicaid/iotagent-json/blob/master/test/unit/HTTP_reveice_measures-test.js). Thus, I'd suggest to remove the `"expression": "${@humedad / 1000}"` from it. It is a simple modification that would works for both NGSIv2 and NGSIv1 (travis will ensure ;) and fix the JSON file.

Reply:
Fixed in e2760b3aee7d517d554809fed0e2f7f1a2102a13